libraries/AP_AHRS/AP_AHRS_View.cpp

```diff
@@ -59,7 +59,7 @@ void AP_AHRS_View::update(bool skip_ins_update)
     rot_body_to_ned = ahrs.get_rotation_body_to_ned();
     gyro = ahrs.get_gyro();
 
-    if (!is_zero(y_angle)) {
+    if (!is_zero(y_angle + _pitch_trim_deg)) {
```

Comment:
This would be less confusing if the test were 
!is_zero(y_angle) || !is_zero(_pitch_trim_deg)
since trim should be limited to < 90 degrees and the point of this is just to save cpu cycles if rot_view is the identity matrix.
Also, if trim is non-zero, won't gyro.rotate(rotation) be wrong too?

Reply:
hum, i think your right, gryo.rotate is wrong with none zero trim, i will see if i can fix

Reply:
i think just need to put rot_view in place of rotation for the gyro. swapping the if round means in the unlikely case that the y_angle = -_pitch_trim_deg we would unnecessarily recalculate.